pkg/controller/inference/playground_controller.go

```diff
@@ -341,14 +359,16 @@ func setPlaygroundCondition(playground *inferenceapi.Playground, service *infere
 			Reason:  "PlaygroundInProgress",
 			Message: "Playground is progressing",
 		}
-		apimeta.SetStatusCondition(&playground.Status.Conditions, condition)
+		changed = apimeta.SetStatusCondition(&playground.Status.Conditions, condition) || changed
 
 		// Set the available to false
 		new_condition := metav1.Condition{
 			Type:   inferenceapi.PlaygroundAvailable,
 			Status: metav1.ConditionFalse,
+			Reason: "PlaygroundNotReady",
```

Comment:
`PlaygroundAvailable=false` implies the reason, so maybe ServiceNotAvailable better?

Reply:
Added the details to the message: `Waiting for inference Service ready`.